### PR TITLE
Blog post setup

### PR DESCRIPTION
**Pull request by chloepouprom**
A lots of files were automatically generated (html, javascript). You can ignore those. The only files to review are:

- `about.Rmd`: https://github.com/LKS-CHART/blog/pull/1/files#r863921690
- `README.md`: https://github.com/LKS-CHART/blog/pull/1/files#r863921563
- `pull_request_template.md`: https://github.com/LKS-CHART/blog/pull/1/files#r863921492

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Summary
```
Comment (chloepouprom) [inline .github/pull_request_template.md:1]: To review

File: README.md [hunk 2]
```diff
@@ -1,2 +1,45 @@
-# blog
-DSAA Blog
+# AA Blog
```
Comment (chloepouprom) [inline README.md:1]: To review

File: about.Rmd [hunk 3]
```diff
@@ -1,9 +1,27 @@
 ---
```
Comment (chloepouprom) [inline about.Rmd:1]: To review

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+# Checklist
+
+[ ] I can successfully knit the Rmd file.
+
+[ ] Figures, graphs, tables render properly in the Rmd file.
+
+[ ] Abbreviations are expanded/explained.
+
+[ ] The Rmd file header contains the post title, post description, author information, and categories.
```
Comment (meggie-d) [inline .github/pull_request_template.md:13]: Wondering if we should add a checklist item for making sure there is no PHI included in the blog post?

Comment (chloepouprom) [reply .github/pull_request_template.md:13]: Yes, good call! I'll update to include that.

**Review by meggie-d — APPROVED** [review]
Looks great, thank you!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,51 @@
...
+
+# Contribute to the blog
+
+- Clone this repo
+
+- Activate the renv environment: `renv::activate()`
+
+- Start a new branch
+
+- Create a new post: `distill::create_post("TITLE OF YOUR BLOG POST", draf = TRUE)` This will create a new folder in the `_posts` folder. This is where you will work on your blog post.
```
Comment (derekbeaton) [inline README.md:17]: change to `draft = TRUE`

File: README.md [hunk 6]
```diff
@@ -1,2 +1,51 @@
...
+
+- Merge the PR when it's approved.
+
+## Blog team
+
+- It's the blog team's responsibility to publish the blog post(s) according to schedule.
+    
+- When it's time to release a new blog post, create a new PR with the following changes: 
+
+    - In the Rmd file, update `draft: true` to `draft: false`.
```
Comment (derekbeaton) [inline README.md:47]: in which Rmd file? the about or the readme? or a different one?

**Review by derekbeaton — COMMENTED** [review]
Just two small comments have been added, and are easily fixed. Feel free to make the changes and approve/merge without another PR

File: README.md (hunk 6 above)
Comment (chloepouprom) [reply README.md:47]: the Rmd that gets created after running the `distill:create_post` command. I'll clarify this and specify

**Pull request merged**